### ISSUE TITLE
Download Pretrained Weight without baidu

### ISSUE THREAD

**Issue by gjustin40** [h751410234/RemoteSensingTeacher#19]
Thank you for providing such valuable information!
I found your paper very inspiring.

Would it be possible to receive the pretrained weights through a method other than Baidu?
Since I am not in China, I am unable to access Baidu.

Could you provide the weights via Google Drive or Gmail?
This is my gmail
gjustin4040@gmail.com

It would be great if I could receive all four models, but if there are any restrictions, I would appreciate at least the xView to DOTA model.

Thank you!

**Comment by h751410234** [comment]
https://drive.google.com/file/d/1opj1ShUrN8XKPq-9RxCrG8LtGiYEgCIG/view?usp=drive_link

**Comment by gjustin40** [comment]
Thank you for your response!
I have requested access via Google Drive.

By the way, i have a question regarding model training.

The Remote Sensing Teacher model does not require labels for the target dataset.
Then why does DAOD.py have an option to include annotations for the target domain as well?
(I understand that xView is the source and DOTA is the target.)
```
        'xView': {
            'train_img': '',
            'train_anno': '',
        },
        'DOTA': {
            'train_img': '',
            'train_anno': '', #### Target domain annotations?
            'val_img': '',
            'val_anno': '',
        },
```

**Comment by h751410234** [comment]
n order to facilitate reading additional dataset information, such as input image index and size, this is necessary for formatting the pseudo-labels into a training format. The category and boxes information are not used. In practice, you could also rewrite the dataloader and pseudo-label generation code to avoid using target domain annotations.

**Comment by h751410234** [comment]
In order to facilitate reading additional dataset information, such as input image index and size, this is necessary for formatting the pseudo-labels into a training format. The category and boxes information are not used. In practice, you could also rewrite the dataloader and pseudo-label generation code to avoid using target domain annotations.

**Comment by gjustin40** [comment]
Thank you for your response!
I think I have understood most of RSTeacher.

One more thing, is there any code available to extract AP performance for each class?

**Comment by h751410234** [comment]
Each type of metric requires modifying the official COCO evaluation script, specifically the installed pycocotools. You should be able to find relevant modification tutorials by searching online.

**Comment by gjustin40** [comment]
Thanks!
Lastly, could yo approve my request about Remote Sensing Teacher pretrained weight from google drive?

**Comment by h751410234** [comment]
I have already agreed.
